11-payment-encoding.md

```diff
@@ -278,6 +281,35 @@ The field is big-endian.  The least-significant bit is numbered 0,
 which is _even_, and the next most significant bit is numbered 1,
 which is _odd_.
 
+Features are listed in [09-features.md](BOLT #9) [FIXME: Or will be after
+flattened features is merged!].
+
+Note that the `var_onion_optin` feature is slightly overloaded to indicate
+support for the `s` field: making this compulsory prevents probing attacks
```

Comment:
would be in favor of not overloading `var_onion_optin` with the `s` field. there's a huge gap between adding TLV onion and actually being able to implement this proposal, perhaps they should be separate?

Reply:
There's a huge gap between implementing `var_onion_optin` and implementing MPP. But implementing support for payment secrets is really simple, right?
So making `var_onion_optin` compulsory to imply that you need to send the payment secret (potentially for existing single-part payments) seems acceptable, even though I'd prefer a dedicated feature bit to make it more explicit (but at the cost of more feature bits in the system...).